spec/model_spec.rb

```diff
@@ -77,6 +81,20 @@ module CryptKeeper
         data = subject.create!(storage: 1)
         data.reload.storage.should == "1"
       end
+
+      it "does not decrypt when stubbing is enabled" do
+        CryptKeeper.stub_encryption = true
+        record = subject.create!(storage: "testing")
+        CryptKeeper::Provider::Encryptor.any_instance.should_not_receive(:decrypt)
+        subject.find(record.id).storage
```

Comment:
rather than using `should_not_receive`, how about just asserting that the plaintext of `storage` is unchanged.